cluster/saltbase/salt/docker/init.sls

```diff
@@ -195,9 +197,22 @@ net.ipv4.ip_forward:
 #    shasum <deb>
 # 6. Update override_deb, override_deb_sha1, override_docker_ver with new
 #    deb name, new hash and new version
+#    For example, for docker 1.8.3-0~wheezy_amd64.deb:
+#        docker_pkg_name - docker-engine
+#        override_deb - docker-engine_1.8.3-0~wheezy_amd64.deb
+#        override_deb_sha1 - f812e1134e7375b29886a372a6e24d04e1ee33c9
+#        override_docker_ver - 1.8.3-0~wheezy
+# 7. Sometimes debian package may not work, to use static linked binary version:
+#    * Find url of specific binary version at:
+#        https://github.com/docker/docker/releases
+#    * Download docker binary and upload to GCS, just like what we do for debian package.
+#    * Update override_bin_name with new binary name. For example, for docker 1.8.3:
+#        override_bin_url - docker-1.8.3
```

Comment:
If both override_deb and override_bin_name are set, the latter will be used. Is that correct?